.github/workflows/tests.yml

```diff
@@ -5,7 +5,7 @@ on:
   push:
 
 env:
-  KONG_VERSION: master
+  KONG_VERSION: disable-h2-alpn-re
```

Comment:
for test?

Reply:
for build with patch https://github.com/Kong/kong/pull/13709 in CI test action.